ryanoasis/nerd-fonts
# Update FiraCode to v3.1

#### Description

_Please explain the changes you made here._

#### Requirements / Checklist

  - If any of the scripts were modified they have been tested and execute without error, e.g.:
    - `./font-patcher Inconsolata.otf --fontawesome --octicons --pomicons`
    - `./gotta-patch-em-all-font-patcher\!.sh Hermit`

#### What does this Pull Request (PR) do?

This PR adds patched and unpatched FiraCode v.3.1 files to the repository

#### How should this be manually tested?

1. Install Fira Code Regular Nerd Font Complete
2. Configure your editor or terminal of choice to use `FiraCode Nerd Font`
3. Enable ligatures if possible
4. Check if you can see the adjusted combinations in your editor or terminal as they are shown here: https://github.com/tonsky/FiraCode#whats-in-the-box

#### Any background context you can provide?

#### What are the relevant tickets (if any)?

#### Screenshots (if appropriate or helpful)
